cmd/etcd-shield/main.go

```diff
@@ -43,6 +44,18 @@ func namespace() string {
 	return os.Getenv("NAMESPACE")
 }
 
+func stateManager(cli client.Client, config *shield.Config) shield.StateManager {
```

Comment:
What if it's `inmemory`? Should we at least log that the value is not recognized and we'll use the default? Debugging to identify this kind of typos will be way easier to me

Reply:
I'll add some logging on which manager we're using.

Reply:
Fixed in 367f047.